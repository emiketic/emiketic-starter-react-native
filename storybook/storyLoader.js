// Auto-generated file created by react-native-storybook-loader
// Do not edit.
//
// https://github.com/elderfo/react-native-storybook-loader.git

function loadStories() {
  require('../src/features/Auth/LoginScreen/LoginScreen.stories');
  require('../src/features/Home/HomeScreen.stories');
}

const stories = [
  '../src/features/Auth/LoginScreen/LoginScreen.stories',
  '../src/features/Home/HomeScreen.stories',
];

module.exports = {
  loadStories,
  stories,
};
