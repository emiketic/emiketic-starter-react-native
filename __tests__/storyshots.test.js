import initStoryshots from '@storybook/addon-storyshots';
import { loadStories } from '../storybook/storyLoader';

initStoryshots({
  framework: 'react-native',
  configPath: './storybook',
  config: ({ configure }) => configure(() => loadStories(), module),
});
