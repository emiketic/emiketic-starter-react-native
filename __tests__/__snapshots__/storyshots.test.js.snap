// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots HomeScreen default 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      appearance="default"
      style={
        Array [
          Object {
            "backgroundColor": "#1F1F1F",
            "height": 1,
          },
          undefined,
        ]
      }
    />
    <View
      appearance="default"
      level="1"
      style={
        Array [
          Object {
            "backgroundColor": "#383838",
          },
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          },
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#3366FF",
            "borderColor": "#3366FF",
            "borderRadius": 4,
            "borderWidth": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "minHeight": 40,
            "minWidth": 40,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 12,
          }
        }
      >
        <Text
          appearance="default"
          category="p1"
          style={
            Array [
              Object {
                "color": "#D4D4D4",
                "fontFamily": "OpenSans-Regular",
                "fontSize": 15,
                "fontWeight": "400",
              },
              Object {
                "color": "#FFFFFF",
                "fontFamily": "OpenSans-Regular",
                "fontSize": 14,
                "fontWeight": "bold",
                "marginHorizontal": 10,
              },
            ]
          }
        >
          <Text
            appearance="default"
            category="p1"
            style={
              Array [
                Object {
                  "color": "#D4D4D4",
                  "fontFamily": "OpenSans-Regular",
                  "fontSize": 15,
                  "fontWeight": "400",
                },
                undefined,
              ]
            }
          >
            Open details
          </Text>
        </Text>
      </View>
    </View>
  </RCTSafeAreaView>
</View>
`;

exports[`Storyshots LoginScreen default 1`] = `null`;
